dashboard/test/ui/features/gdpr_dialog.feature

```diff
@@ -0,0 +1,36 @@
+@dashboard_db_access
+
+Feature: GDPR Dialog - data transfer agreement
+
+  Scenario: EU user sees the GDPR Dialog on dashboard, opt out
+    Given I am a teacher
+    Given I am on "http://studio.code.org/home?force_in_eu=1"
+    Then I wait to see ".ui-test-gdpr-dialog"
+    Then I wait to see ".ui-test-gdpr-dialog-logout"
+    Then I click selector ".ui-test-gdpr-dialog-logout"
```

Comment:
Another future cleanup idea:  I like building domain-specific steps so that the UI tests read like English.  Let's add a file `dashboard/test/ui/features/step_definitions/gdpr.rb` that defines steps so these tests can read like:

```gherkin
Scenario: EU user sees the GDPR Dialog on dashboard, opt out
  Given I am a teacher
  And I am located in the EU
  And I am on "http://studio.code.org/home"
  And I wait to see the GDPR data transfer dialog
  When I click the GDPR logout button
  Then check that I am on "http://code.org/"
  And I do not see the GDPR data transfer dialog
```